### DISCUSSION TITLE
micro frontends support other than Webpack module federation?

### DISCUSSION THREAD

**Discussion by semanticist21** [nrwl/nx#27499] [Feature Requests]
Hi, I'm using Webpack module federation in my project.
I found that webpack is extremely slow when hmr is applied..

Are there any plants to integrate other micro frontends bundler such as Rspack or originjs/vite..?

**Comment by Rezanmonis** [comment]
Hello

We are looking for frontend and backend developers as our teammates for our edtech project rezanmonis@gmail.com
